### PR TITLE
Pagination Added to ListAccounts Validator RPC Call

### PR DESCRIPTION
**Pull request by rauljordan**
We definitely need to paginate the accounts as we will display them in an Angular material list in the UI. This PR adds pagination capabilities

### CODE REVIEW

**Review by farazdagi — APPROVED** [review]
Looks great, absolutely loved the unit test (concise, yet well covering).

File: proto/validator/accounts/v2/web_api.proto [hunk 1]
```diff
@@ -154,10 +154,30 @@ message WalletResponse {
...
 message ListAccountsResponse {
     repeated Account accounts = 1;
+
+    // A pagination token returned from a previous call to `ListAccounts`
+    // that indicates from where listing should continue.
+    // This field is optional.
+    string next_page_token = 2;
+
+    // Total count matching the request.
+    int32 total_size = 3;
```
Comment (prestonvanloon) [inline proto/validator/accounts/v2/web_api.proto:180]: uint64 or do you plan on using negative numbers?

Comment (farazdagi) [reply proto/validator/accounts/v2/web_api.proto:180]: I also had this question, but after looking into other paginated entities (like `ListValidatorBalancesRequest`) and at `shared/pagination/paginator.go` itself (which expects `int` as page size), I supposed it was like that for consistency. Was it (not sure now)?

Comment (rauljordan) [reply proto/validator/accounts/v2/web_api.proto:180]: we've been using int32 always, so it was just for consistency. Can change them all in a separate PR

**Pull request merged**